### ISSUE TITLE
Bug when update calibreweb

### ISSUE THREAD

**Issue by DoumeDuSud** [YunoHost-Apps/calibreweb_ynh#38]
https://paste.yunohost.org/raw/ugoqewojem

Apparently is about pillow library

```
Attention :     Please see the install instructions at:
Attention :        https://pillow.readthedocs.io/en/latest/installation.html
Attention : 
Attention : 
Attention :     ----------------------------------------
Attention : ERROR: Command errored out with exit status 1: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-vdk6cp1d/Pillow/setup.py'"'"'; __file__='"'"'/tmp/pip-install-vdk6cp1d/Pillow/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-p9s6n9ab/install-record.txt --single-version-externally-managed --home /tmp/pip-target-dvtb_kqo --compile --install-headers /tmp/pip-target-dvtb_kqo/include/python/Pillow Check the logs for full command output.
Attention : [ERR] Upgrade failed.
```

**Comment by Krakinou** [comment]
Hmmm,
on which board/VM type are you? I have quite a lot of issue regarding pip dependencies that did not showed up when I tested the package... 

Can you try `apt install libjpeg-dev` and try again?
If it solved the issue I'll add it to the dependencies list.

**Comment by Krakinou** [comment]
Solved by #39 according to https://forum.yunohost.org/t/mise-a-jour-de-calibreweb/11340/12

**Comment by DoumeDuSud** [comment]
Thanks :1st_place_medal:
